Packs/ApiModules/ReleaseNotes/2_2_30.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### TAXII2ApiModule
+
+- Updated the parsing logic for the autonomous system indicator. When the indicator value will be a number, it will be converted to a string.
```

Comment:
```suggestion
- Updated the parsing logic for the autonomous system indicator. When the indicator value is a number, it will be converted to a string.
```